controllers/auth.controller.js

```diff
@@ -33,13 +33,13 @@ const signUp = async (req, res, next) => {
       hashPassword
     );
 
-    try {
-      await transporter.sendMail(getMailConfigurations(email, token));
-    } catch (err) {
-      return res
-        .status(400)
-        .json({ error: "Unable to send email", field: "email" });
-    }
+    // try {
+    //   await transporter.sendMail(getMailConfigurations(email, token));
+    // } catch (err) {
+    //   return res
+    //     .status(400)
+    //     .json({ error: "Unable to send email", field: "email" });
+    // }
```

Comment:
Why do we need this? Or is this previous code?

Reply:
This was previous code used to send emails. However, I was not able to make nodemailer work because of a problem with credentials I think, so commented it out.